Android/ZensoftHRCRM/app/src/main/java/com/erkprog/zensofthrcrm/data/network/interviews/InterviewsRepository.java

```diff
@@ -36,8 +36,30 @@ public void onFailure(Call<InterviewsResponse> call, Throwable t) {
     });
   }
 
+  @Override
+  public void getInterviewDetails(final InterviewDetailContract.Repository.OnFinishedListener
+                                      onFinishedListener, Context mContext) {
+
+    Call<Interview> call = RestClientTest.getClient(mContext).getDetailedInterview();
+
+    call.enqueue(new Callback<Interview>() {
+      @Override
+      public void onResponse(Call<Interview> call, Response<Interview> response) {
+        Interview body = response.body();
```

Comment:
We need to only if response.isSuccessful and response.body != null
Further data need to check when we use it

Reply:
it was unused file. Deleted now